pytest_inmanta/__init__.py

```diff
@@ -15,3 +15,26 @@
 
     Contact: code@inmanta.com
 """
+from typing import Optional
+
+import pkg_resources
+from pkg_resources import DistributionNotFound
+
+from packaging import version
+
+CORE_VERSION: Optional[version.Version]
```

Comment:
In the test code I would import this file instead of doing it over there.

FYI: inmanta/inmanta-core#6175